core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -122,7 +126,7 @@ public PagedDoraWorker(AtomicReference<Long> workerId, AlluxioConfiguration conf
         .build(new CacheLoader<String, DoraMeta.FileStatus>() {
           @Override
           public DoraMeta.FileStatus load(String path) throws IOException {
-            UfsStatus status = ufs.getStatus(path);
+            UfsStatus status = mUfs.getStatus(path);
             DoraMeta.FileStatus fs = buildFileStatusFromUfsStatus(status, path);
             return fs;
```

Comment:
inline? 
return buildFileStatusFromUfsStatus(mUfs.getStatus(path), path);

Reply:
I am not so familiar with the Java programming language.
IMHO, making it inline does not improve performance, nor less GC.
Then, why do I write the code like this?
Because when I do live debugging, it is easier for me to inspect the value of every variable and the return value from a function.

Maybe we can discuss this programming convention.